internal/controller/aigateway_controller.go

```diff
@@ -255,7 +256,8 @@ func (r *AiGatewayReconciler) generateAiGatewayConfig(ctx context.Context, aiGat
 	config := LiteLLMConfig{
 		ModelList: modelList,
 		LiteLLMSettings: LiteLLMSettings{
-			RequestTimeout: DefaultRequestTimeout, // 10 minutes default timeout
+			RequestTimeout:  DefaultRequestTimeout, // 10 minutes default timeout
+			SuccessCallback: []string{"otel"},
```

Comment:
What's this for? Please add a description.

Btw: Please cleanup the comment above, which uses a hard-coded number for a variable.